src/Tools/VMC/EvtLibInteractionListGenerator.cxx

```diff
@@ -0,0 +1,62 @@
+#include "Tools/VMC/EvtLibInteractionListGenerator.h"
+
+#include "Framework/EventGen/InteractionList.h"
+#include "Framework/Interaction/Interaction.h"
+#include "Framework/Messenger/Messenger.h"
+#include "Framework/ParticleData/PDGCodes.h"
+#include "Framework/ParticleData/PDGUtils.h"
+
+using namespace genie;
+using namespace genie::vmc;
+
+//___________________________________________________________________________
+EvtLibInteractionListGenerator::EvtLibInteractionListGenerator() :
+InteractionListGeneratorI("genie::vmc::EvtLibInteractionListGenerator")
+{
+
+}
+//___________________________________________________________________________
+EvtLibInteractionListGenerator::EvtLibInteractionListGenerator(string config):
+InteractionListGeneratorI("genie::vmc::EvtLibInteractionListGenerator",  config)
+{
+
+}
+//___________________________________________________________________________
+EvtLibInteractionListGenerator::~EvtLibInteractionListGenerator()
+{
+
+}
+//___________________________________________________________________________
+InteractionList * EvtLibInteractionListGenerator::CreateInteractionList(
+   const InitialState & init_state) const
+{
+  InteractionList * intlist = new InteractionList;
+
+  // REVIEW these are all these arbitrary choices, but seems to be required to
+  // be able to succesfully pick a spline. Perhaps it would be better to add
+  // new enum entries for "unspecified because it came from an external
+  // library"?
+
```

Comment:
Yes, rather thane using `kScQuasiEleastic` we should add a new "scattering type" for external generators.  In my parallel start on something similar (but completely imcomplete), I added ` kScExternalGen = 999`, see
https://github.com/nusense/Generator/blob/extgen/src/Framework/Interaction/ScatteringType.h#L55

Reply:
Done